src/index.ts

```diff
@@ -0,0 +1,24 @@
+import dotenv from 'dotenv';
+dotenv.config();
+import express from 'express';
+import bodyParser from 'body-parser';
+import cors from 'cors';
+
+import zdRoutes from './restapi/zd_routes';
+import mmRoutes from './restapi/mm_routes';
+
+const app = express();
+app.use(cors({origin: 'http://localhost:8065', credentials: true}));
+
+app.use(bodyParser.json());
+app.use(bodyParser.urlencoded({
+    extended: true,
+}));
+
+app.use('/zendesk', zdRoutes); // Zendesk router
+
+// Mattermost routes must be at root URL level.  The cloud-apps proxy plugin
+// resolves certain paths based on the provided Root URL
+app.use('', mmRoutes);
```

Comment:
> Mattermost routes must be at root URL level

Why not mount at `/mattermost` and have the manifest say that `http://localhost:4000/mattermost` is the App's root URL?

Reply:
Simple solution.  Tested, works, and committed. https://github.com/mattermost/mattermost-app-zendesk/commit/c50f74a0170784181b1c3dc4e421e0a702085bfc